README.md

```diff
@@ -1,15 +1,162 @@
-Quire
-=====
-multi-format digital publishing
--------------------------------
+![Quire logo](static/img/quire_logo_tm.png)
 
-This is the mono-repository for [Quire](https://quire.getty.edu), is a multi-format publishing framework. This multi-package repository contains:
+## About Quire 
 
-- `packages/cli`: the `quire` command-line interface
-- `starters/default`: a default starter project 
-- `themes/default`: the default publication theme
+Developed by Getty, Quire is a new, multiformat publishing tool available for immediate access and use. Quire is in a limited beta, © J. Paul Getty Trust, and not yet released as open-source software. **For a free license to use Quire, please complete [this form](http://bit.ly/quire-signup).**
 
+Quire is an ideal tool for anyone seeking to publish a beautiful, scholarly digital book optimized for visual imagery and designed to ensure its content is widely accessible and stable. 
 
-### License
+Quire books feature:
+
+- Page-level citation
+- Footnotes
+- Bibliographies
+- Figure images and image groups
+- Zooming images and maps
+- Video and audio embeds
+- Dynamic tables of contents and menus
+- Full-text search
+- Responsive web design
+- Web accessibility and SEO optimized
+- Outputs to web, PDF, and EPUB formats
+
+To learn more about Quire, please refere to our in progress [documentation](https://quire.getty.edu)
+
+Check out [the demo site](https://thegetty.github.io/quire-starter/), and [some of Getty’s books developed with Quire](http://www.getty.edu/publications/digital/digitalpubs.html)
```

Comment:
I need to fix the font pathing on the demo site.

But @mphstudios, what do you think we should we do about hosting the demo site going forward? We definitely need it but it's currently hosted as a gh-pages site from the now-defunct quire-starter repo. Maybe ideally it would be integrated into the main Quire website at https://quire.getty.edu/demo/ or something like that?